frontend/src/App.tsx

```diff
@@ -91,11 +91,11 @@ function AppToolbar() {
       </Typography>
       {[EXPLORE_DATA_PAGE_LINK, DATA_CATALOG_PAGE_LINK, ABOUT_US_PAGE_LINK].map(
         (pageUrl, i) => (
-          <Button className={styles.NavButton} key={i}>
-            <LinkWithStickyParams to={pageUrl}>
-              {PAGE_URL_TO_NAMES[pageUrl]}
+            <LinkWithStickyParams to={pageUrl} class={styles.NavLink}>
```

Comment:
What if you put the NavLink class on the Button directly? then you wouldn't have to add the prop to LinkWithStickyParams or nest the CSS

Reply:
Unfortunately, this did not seem to work.  :(  I haven't been able to sleuth it out, but I tried adding the NavLink class to the button and altering CSS with a few variations on:
```.NavLink {
  color: $white;
  &:visited, &:link {
    color: white;
    text-decoration: none;
  }
}```
I've pushed a more condensed version of the previous CSS just to clean up the code a little bit regardless